src/@utils/index.ts

```diff
@@ -17,3 +17,27 @@ export function sortAssets(items: Asset[], sorted: string[]) {
   })
   return items
 }
+
+export const isObjectLike = (object: any) => {
+  return object !== null && typeof object === 'object' && !Array.isArray(object)
+}
+
+export const getObjectPropertyByPath = (object: any, path = '') => {
```

Comment:
nitpick(non-blocking): `path` parameter could be typed as string.

Reply:
implicitly typed as string due to assignment to `''` (specific typing actually also gets removed here by code formatting settings)